Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -189,6 +241,37 @@ public bool TryGetNearGraspPoint(out Vector3 result)
             }
         }
 
+
+        private static readonly ProfilerMarker TryGetNearGraspAxisPerfMarker = new ProfilerMarker("[MRTK] ConePointer.TryGetNearGraspAxis");
+
+        /// <summary>
+        /// Gets the axis that the grasp happens
+        /// For the SpherePointer it's the axis from the palm to the index tip
+        /// For any other IMixedRealityController, return just the pointer's forward orientation
+        /// </summary>
+        public bool TryGetNearGraspAxis(out Vector3 result)
+        {
+            using (TryGetNearGraspAxisPerfMarker.Auto())
+            {
+                // If controller is of kind IMixedRealityHand, return average of index and thumb
```

Comment:
Is this comment correct?

Reply:
It seems like this is returning a lerp between the palm and index finger, not the average of index and thumb.